Sources/CrowdNotifierBaseSDK/Models/Protobufs_v2.pb.swift

```diff
@@ -2,7 +2,7 @@
 // swift-format-ignore-file
```

Comment:
You can add something like this to the `.gitattributes` file:

```
Models/*.pb.swift -diff
```

Then these files won't show up in the github diffs.